src/test/ui/const-generics/invalid-enum.stderr

```diff
@@ -25,70 +25,130 @@ LL |   let _: Example<CompileFlag::A, _> = Example { x: 0 };
    |                  not a type
    |                  help: try using the variant's enum: `CompileFlag`
 
-error[E0107]: wrong number of const arguments: expected 1, found 0
+error[E0107]: wrong number of const arguments for `Example`: expected 1, found 0
   --> $DIR/invalid-enum.rs:31:10
    |
+LL | pub struct Example<const CF: CompileFlag, T=u32>{
+   | ------------------------------------------------ defined here, with 1 const parameter: `T`
+...
 LL |   let _: Example<CompileFlag::A, _> = Example { x: 0 };
-   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 1 const argument
+   |          ^^^^^^^ ----------------- found no const argument in here
+   |
+help: add missing const argument:
+   |
+LL |   let _: Example<CompileFlag::A, _, T> = Example { x: 0 };
+   |                                   ^^^
 
-error[E0107]: wrong number of type arguments: expected at most 1, found 2
+error[E0107]: wrong number of type arguments for `Example`: expected at most 1, found 2
   --> $DIR/invalid-enum.rs:31:10
    |
+LL | pub struct Example<const CF: CompileFlag, T=u32>{
+   | ------------------------------------------------ defined here, with at most 1 type parameter: `CF`
+...
 LL |   let _: Example<CompileFlag::A, _> = Example { x: 0 };
-   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected at most 1 type argument
+   |          ^^^^^^^
```

Comment:
We will want a label here, likely "expected at most 1, found 2". If we do that, we could potentially remove that from the main message.

Reply:
this message is not applicable anymore, because the const-mismatched-for-type case is handled elsewhere now (seems like it was changed a month or two ago)